docs/data/data-grid/row-updates/row-updates.md

```diff
@@ -39,6 +39,35 @@ In addition, the area in which `onRowsScrollEnd` is called can be changed using
 
 {{"demo": "InfiniteLoadingGrid.js", "bg": "inline", "disableAd": true}}
 
+## Lazy loading [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+:::warning
+This feature is experimental, it needs to be explicitly activated using the `lazyLoading` experimental feature flag.
+
+```tsx
+<DataGridPro experimentalFeatures={{ lazyLoading: true }} {...otherProps} />
+```
+
+:::
+
+Lazy Loading works like a pagination system, but instead of loading new rows based on pages, it loads them as the user scroll through the grid and reveals empty rows.
```

Comment:
```suggestion
Lazy Loading works like a pagination system, but instead of loading new rows based on pages, it loads them as the user scrolls through the grid and reveals empty rows.
```